x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/edit_package_policy_page/index.tsx

```diff
@@ -714,7 +714,7 @@ const IntegrationsBreadcrumb = memo<{
   policyName: string;
   pkgkey: string;
 }>(({ pkgTitle, policyName, pkgkey }) => {
-  useIntegrationsBreadcrumbs('integration_policy_edit', { policyName, pkgTitle, pkgkey });
+  useIntegrationsBreadcrumbs('integration_policy_upgrade', { policyName, pkgTitle, pkgkey });
```

Comment:
If I navigate from the integration page and edit a policy I got the upgrade integration it is correct? I think we should have the edit breadcrumb in this case

<img width="1409" alt="Screen Shot 2022-02-23 at 3 53 37 PM" src="https://user-images.githubusercontent.com/1336873/155406541-0ac91ce1-04df-40cf-a773-39fa0c348ad3.png">
<img width="853" alt="Screen Shot 2022-02-21 at 2 09 13 PM" src="https://user-images.githubusercontent.com/1336873/155406552-cab72728-7ac4-45bd-944e-ba8adc2390b0.png">

Reply:
No this is not correct - taking a look now.

Reply:
Fixed in https://github.com/elastic/kibana/pull/126295/commits/d66f582a585c9a94bd4313d4756908c95b5a08e7